src/components/visual/side-panel/side-panel.tsx

```diff
@@ -10,10 +9,12 @@ import { SidePanelSideEnum } from './side-panel.types';
 })
 export class SidePanel {
   @Prop() isOpen: boolean = false;
-  @Prop() side: SidePanelSideEnum = SidePanelSideEnum.RIGHT;
   @Prop() panelClassName?: string;
+  @Prop() panelTitle: string;
```

Comment:
title & class ?

Reply:
The `title` keyword is reserved in Stencil, there's a lint or TypeScript warning regarding this. As for the `panelClassName`, we've decided to only use the `class` property on elements where the class is directly passed to the first element in the markup of the component.
<img width="853" alt="image" src="https://github.com/user-attachments/assets/b9cbf185-bdd7-4fdd-98e0-ca7b806c1590" />